monitoring/bentoml-monitoring-arize/pyproject.toml

```diff
@@ -39,11 +39,11 @@ test = ["pytest"]
 Homepage = "https://github.com/bentoml/ecosystem/tree/main/bentoml/bentoml-monitoring-arize"
 
 [tool.hatch.version]
-path = "src/bentoml/monitoring/arize/version.py"
+path = "src/bentoml_monitoring_arize/version.py"
```

Comment:
why do you need to rename this? This will make sure when installing this package it would be 
`bentoml.monitoring.arize` instead of `bentoml_monitoring_arize`

Reply:
Because `bentoml_monitoring_arize` is what I want to have.

Reply:
any specific reason?